config/config_test.go

```diff
@@ -15,26 +15,19 @@
 package config
 
 import (
-	"io"
+	"bytes"
 	"io/ioutil"
 	"os"
 	"path"
 	"reflect"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	log "github.com/sirupsen/logrus"
 
-	"github.com/mendersoftware/mender-connect/client/https"
+	"github.com/stretchr/testify/assert"
```

Comment:
if I read this diff correctly, there is an extra empty line here that is not needed.

Reply:
It seems like. But the space is pre-existing and to have the `testify` packages separated is very common in test files. I'll leave it.